packages/hopp-plugin-concat/lib/index.js

```diff
@@ -18,7 +18,35 @@ export const config = {
 
 /**
  * We don't need to do any real transformation.
+ * 
+ * suffix and prefix features
+ * @param suffix || null @param prefix || null
+ * appending additional suffix and prefix to each data.
```

Comment:
This JSDoc comment is incorrect:

 - Each param definition should be on its own line
 - Description should only be before the param comments start